wg-lts/annual-report-2023.md

```diff
@@ -5,23 +5,23 @@
 
 1. What work did the WG do this year that should be highlighted?
 
-<!--
-   Some example items that might be worth highlighting:
-   - artifacts
-   - reports
-   - white papers
-   - work not tracked in KEPs
--->
+During 2023, we worked to restart the WG.
+
+We also designed and published a [survey](https://bit.ly/k8s-upgrade-survey) to capture pain points and better understand challenges Kubernetes users face with upgrading. We collected 223 survey responses. From this data, we identified some  areas where we can better communicate improvements that have already been made to the project. We plan to work on several of these during 2024 and will present analyzed results from the survey to the community and sponsoring SIGs during 2024. 
+
+We also hosted a [maintainer track session](https://www.youtube.com/watch?v=0fngdOlwZtQ) at KubeCon NA in Chicago to discuss what the working group wanted to achieve.
+
+We have been meeting on a bi-weekly cadence, but have not provided updates to the sponsoring SIGs as most of 2023 was spent forming and drafting the survey. We will provide updates to the sponsoring SIGS during 2024.   
```

Comment:
nit,
```suggestion
We have been meeting on a bi-weekly cadence, but have not provided updates to the sponsoring SIGs as most of 2023 was spent forming and drafting the survey. We will provide updates to the sponsoring SIG during 2024.
```

Reply:
the change `SIG` is applicable on line 10 too..

Reply:
There are several sponsoring SIGs for this WG.